pkg/yurtctl/cmd/revert/revert.go

```diff
@@ -213,30 +219,18 @@ func (ro *RevertOptions) RunRevert() (err error) {
 		return
 	}
 
-	// 6. recreate the system:controller:node-controller clustrrolebinding
-	ncClusterrolebinding := &rbacv1.ClusterRoleBinding{
-		ObjectMeta: metav1.ObjectMeta{
-			Name: "system:controller:node-controller",
-		},
-		RoleRef: rbacv1.RoleRef{
-			APIGroup: "rbac.authorization.k8s.io",
-			Kind:     "ClusterRole",
-			Name:     "system:controller:node-controller",
-		},
-		Subjects: []rbacv1.Subject{
-			{
-				Kind:      "ServiceAccount",
-				Name:      "node-controller",
-				Namespace: "kube-system",
-			},
+	// 6. enable node-controller
+	if err = kubeutil.RunServantJobs(ro.clientSet,
+		map[string]string{
+			"action":                "enable",
+			"yurtctl_servant_image": ro.YurtctlServantImage,
```

Comment:
why use `ro.YurtctlServantImage` to enable or disable nodelifecycle controller?

Reply:
Because the system architecture may be amd64, arm64, arm, using a unified basic image may cause problems. So the parameter of yurtctl convert --yurtctl-servant-image is reused here.
The yurtctl-servant image may be separated from yurtctl in the future for more elegant use.